modules/pool/detail/components/warning/PoolStakeInFarmWarning.tsx

```diff
@@ -52,5 +52,11 @@ export function PoolStakeInFarmWarning() {
         }
     }, [hasBptInWallet]);
 
+    useEffect(() => {
```

Comment:
Why is this in this PR?

Reply:
I ran into this while testing the PR. It is not related and I will create a separate PR for it.